### ISSUE TITLE
keyboard navigation doesn't work correctly

### ISSUE THREAD

**Issue by alanbuxey** [MyUNiDAYS/competency-framework#73]
after loading the site keyboard navigation cannot be used to select the e.g. Competency and Role options.  If a mouse is used, then tab navigation can jump to chosen tree - but not to eg Role if Competency has the mouse focus.  A sub-tree e.g. Core list can not be opened by keyboard navigation but keyboard can be used to go through the sub-tree (but then not onto other sub-trees)
